lms/djangoapps/grades/new/course_grade.py

```diff
@@ -123,26 +120,42 @@ def compute_and_update(self, read_only=False):
 
         If read_only is True, doesn't save any updates to the grades.
         """
-        self._log_event(log.warning, u"compute_and_update, read_only: {}".format(read_only))
         subsection_grade_factory = SubsectionGradeFactory(self.student, self.course, self.course_structure)
+        subsections_total = 0
+        blocks_total = 0
         for chapter_key in self.course_structure.get_children(self.course.location):
             chapter = self.course_structure[chapter_key]
             chapter_subsection_grades = []
             for subsection_key in self.course_structure.get_children(chapter_key):
-                chapter_subsection_grades.append(
-                    subsection_grade_factory.create(self.course_structure[subsection_key], read_only=True)
+                subsections_total = subsections_total + 1
+                subsection_grade = subsection_grade_factory.create(
+                    self.course_structure[subsection_key],
+                    read_only=True
                 )
+                chapter_subsection_grades.append(subsection_grade)
+                blocks_total = blocks_total + len(subsection_grade.locations_to_weighted_scores)
 
             self.chapter_grades.append({
                 'display_name': block_metadata_utils.display_name_with_default_escaped(chapter),
                 'url_name': block_metadata_utils.url_name_for_block(chapter),
                 'sections': chapter_subsection_grades
             })
 
+        subsections_read = len(subsection_grade_factory._unsaved_subsection_grades)  # pylint: disable=protected-access
+        subsections_created = subsections_total - subsections_read
         if not read_only:
             subsection_grade_factory.bulk_create_unsaved()
 
         self._signal_listeners_when_grade_computed()
+        self._log_event(
+            log.warning,
+            u"compute_and_update, read_only: {0}, {1}/{2} subsections read/created, {3} blocks accessed".format(
```

Comment:
Nit: to stay consistent with the formatting that @sanfordstudent had introduced, put the identifier in the front with a colon before the value.  So it would be:
"compute_and_update, read_only: {0}, subsections read/created: {1}/{2} , blocks accessed: {3}"